db/init.go

```diff
@@ -80,7 +80,8 @@ func GetData(rows *sql.Rows) ([]string, [][]string, error) {
 				case "BIGINT":
 					value = fmt.Sprintf("%s", col)
 				case "INT":
-					value = fmt.Sprintf("%d", col)
+					//value = fmt.Sprintf("%d", col)
+					value = fmt.Sprintf("%s", col)
```

Comment:
We actually are unsure about this.

In the initial implementation, `%d` was sometimes returning `[]` instead of `0`.

But as you declare `value` as being a string anyways: https://github.com/lefred/innotopgo/blob/d3676bcf305a79637a02893b520f55fcff991a95/db/init.go#L73
I guess `%s` should do the trick. At least we always got the expected results with it

Reply:
Thank you for the addition !